src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs

```diff
@@ -65,22 +66,27 @@ public FileStream(IntPtr handle, FileAccess access, bool ownsHandle, int bufferS
         private static void ValidateHandle(SafeFileHandle handle, FileAccess access, int bufferSize, bool isAsync)
         {
             if (handle.IsInvalid)
+            {
                 throw new ArgumentException(SR.Arg_InvalidHandle, nameof(handle));
-
-            if (access < FileAccess.Read || access > FileAccess.ReadWrite)
+            }
+            else if (access < FileAccess.Read || access > FileAccess.ReadWrite)
```

Comment:
What is this `if` to `if-else` change achieving OR is there a guideline suggesting this? I failed to find it.

Reply:
It's an old habit that comes from https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1513.md

![obraz](https://user-images.githubusercontent.com/6011991/111622048-04a53e00-87e9-11eb-9fe2-edacc81f4ca9.png)